<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fenetreequipe</class>
 <widget class="QDialog" name="Fenetreequipe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion Equipes</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Choix de l'equipe</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="SuppresionBox"/>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="ButtonEquipe">
     <property name="text">
      <string>Ajouter/supprimer Equipe</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Joueurs </string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tableWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="ButtonAjouter">
          <property name="text">
           <string>Ajouter Joueur</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="ButtonSupprimer">
          <property name="text">
           <string>Supprimer Joueur</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="ButtonModifier">
          <property name="text">
           <string>Modifier Joueur</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="ButtonAnnuler">
     <property name="text">
      <string>Valider Equipe</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_Import">
     <property name="text">
      <string>Importer Equipe</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton_Export">
     <property name="text">
      <string>Exporter Equipe</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
